app/screens/timezone/index.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
2015

Reply:
I'm going to look into adding the automatic license header check from the web app this afternoon to save us all some effort. Scrolling past these is hard :p